### PR TITLE
Fix BgTask Description and minor refactoring.

### LINKED ISSUE #13429: BackgroundTask Description is no more editable

### PR DESCRIPTION
**Pull request by jtkech**
Fixes #13429 

Before working more in separate PRs on what is missing around Background Tasks/Jobs.

Here some little refactoring I'm working on first.

Doing so I saw a little issue, the BgTask Description is no more editable, which is also fixed here.

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.BackgroundTasks/Views/BackgroundTask/Edit.cshtml [hunk 1]
```diff
@@ -29,7 +29,7 @@
             <div class="@Orchard.GetWrapperCssClasses()" asp-validation-class-for="Description">
                 <label asp-for="Description" class="@Orchard.GetLabelCssClasses()">@T["Description"]</label>
                 <div class="@Orchard.GetEndCssClasses()">
-                    <input asp-for="Description" class="form-control-plaintext" readonly />
```
Comment (hishamco) [inline src/OrchardCore.Modules/OrchardCore.BackgroundTasks/Views/BackgroundTask/Edit.cshtml:32]: Oops, why it was readonly?!!

File: src/OrchardCore/OrchardCore.Abstractions/BackgroundTasks/BackgroundTaskExtensions.cs [hunk 2]
```diff
@@ -12,41 +12,30 @@ public static BackgroundTaskSettings GetDefaultSettings(this IBackgroundTask tas
...
-                    LockExpiration = attribute.LockExpiration
+                    LockExpiration = attribute.LockExpiration,
                 };
             }
 
             return new BackgroundTaskSettings()
             {
                 Name = technicalName,
-                Title = technicalName
+                Title = technicalName,
```
Comment (hishamco) [inline src/OrchardCore/OrchardCore.Abstractions/BackgroundTasks/BackgroundTaskExtensions.cs:32]: Still I'm wondering why you adding a comma at the end :)

Comment (jtkech) [reply src/OrchardCore/OrchardCore.Abstractions/BackgroundTasks/BackgroundTaskExtensions.cs:32]: This is an analyzer rule we are using in some projects, for example it is easier to re-order the lines without dealing with the commas, even if we move the last line, I like it.

https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1413.md

**Comment by sebastienros** [comment]
Description can be changed but not Title? Was is editable before and why?

**Comment by jtkech** [comment]
@sebastienros 

> Description can be changed but not Title? Was is editable before and why?

No problem, here we only make Description editable as it was before.
And Title, that was introduced recently, is still not editable.

**Comment by MikeAlhayek** [comment]
@jtkech since this is a follow up PR to another was merged in 1.6, you may want to merge this ASAP. Then merge `main` into `release/1.6` to ensure this change take it in the 1.6 release which should be released tomorrow during triage.

**Comment by jtkech** [comment]
Merged to main, I let others decide if it should be part of 1.6

**Pull request merged**